### PR TITLE
Add instructions for playing guitar

### PR DESCRIPTION
**Pull request by dishangupta**
Please review this, @casey-collab

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: english/guitar/playing-guitar/guitar-lesson-1.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Identify the parts of the guitar. Whether you're playing an electric or an acoustic guitar, the instrument is
```
Comment (casey-collab) [inline english/guitar/playing-guitar/guitar-lesson-1.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**